src/Microsoft.Azure.SignalR.Management/HubInstanceFactories/MultiEndpointConnectionContainerFactory.cs

```diff
@@ -27,31 +24,16 @@ public MultiEndpointConnectionContainerFactory(IServiceConnectionFactory connect
             _router = router;
         }
 
-        public MultiEndpointServiceConnectionContainer GetOrCreate(string hubName, ILoggerFactory loggerFactoryPerHub = null)
+        public MultiEndpointServiceConnectionContainer CreateAndStart(string hubName)
```

Comment:
naming: ConnectAsync? e,g, Microsoft.Azure.SignalR.ConnectionFactory

Reply:
It is a container factory not a connection factory?

Reply:
connection vs connectionContainer: only 1 vs many?